deepchem/models/torch_models/layers.py

```diff
@@ -6830,6 +6830,371 @@ def forward(self, x: torch.Tensor,
 
         return x, coords
 
+def _DAGgraph_step(
```

Comment:
Add a docstring please! (Even though this is an internal function we should still have a docstring and usage example)